arviz/stats/stats_utils.py

```diff
@@ -123,7 +123,15 @@ def _ufunc(*args, out=None, **kwargs):
                 msg += " Correct shape is {}".format(arys[-1].shape[:-n_dims])
                 raise TypeError(msg)
         for idx in np.ndindex(out.shape):
-            arys_idx = [ary[idx].ravel() if ravel else ary[idx] for ary in arys]
+            try:

```

Comment:
do we need try except here? Could this all be done with

                arys_idx = []
                for ary in arys:
                    idx_broadcast = tuple(
                        0 if dim_len == 1 else idx_i for idx_i, dim_len in zip(idx, ary.shape)
                    )
                    arys_idx.append(ary[idx_broadcast].ravel() if ravel else ary[idx_broadcast])